app/controllers/api/v1/teams_controller.rb

```diff
@@ -34,12 +32,7 @@ def update
     end
   end
 
-  def shift_availabilities
-    validate_shift_params
-    data = team_availability(@day, @s, @e)
```

Comment:
minor nit but use more descriptive name. I have no clue what @s or @e are

Reply:
No, you are right, but this is a function that has been removed. Probably best to do the review locally, too, so you can notice this.

Reply:
This is actually very bad on my end as I had intended to change it to `@start` and `@end` after testing or at least adding comments. Fortunately, it is gone now.